.bazelrc

```diff
@@ -0,0 +1,9 @@
+build --action_env CC=clang
+build --action_env CXX=clang++
```

Comment:
Nit: be consistent about equals sign between flag name and flag value. Below you have it, here you don't.

Reply:
Done!